### PR TITLE
TNS buttons

### PR DESCRIPTION
**Pull request by griffin-h**
We should have buttons on the TOM to (1) report new transients to the TNS and (2) report classifications of existing transients to the TNS. I have implemented the first one so far. Currently it uses the TNS sandbox until we are ready to actually start reporting things. There is still a problem with getting the report status back from the TNS. I have an email out to Ofer Yaron about this.

### CODE REVIEW

**Comment by griffin-h** [comment]
I've implemented both buttons now. The "report" button appears on the target detail page if the target name starts with neither "AT" nor "SN." The "classify" button appears if the target name starts with "AT." The names are updated in response to the reports.

File: custom_code/forms.py [hunk 1]
```diff
@@ -2,7 +2,276 @@
...
+        (1, "PSN"),
+        (2, "PNV"),
+        (3, "AGN"),
+        (4, "NUC"),
+        (5, "FRB"),
+    ], initial=(1, "PSN"))
+    non_detection__archiveid = forms.ChoiceField(choices=[
+        (0, "Other"),
+        (1, "SDSS"),
+        (2, "DSS"),
```
Comment (jrastinejad) [inline custom_code/forms.py:126]: Should this be CSS?

**Review by jrastinejad — APPROVED** [review]
Looks great! One possible typo (DSS --> CSS ?)

File: custom_code/forms.py (hunk 1 above)
Comment (griffin-h) [reply custom_code/forms.py:126]: It actually is DSS (Digitized Sky Survey)! We may eventually want to ask them to add more options, but for now I just included what the TNS currently has.

**Pull request merged**